src/oc_reporter_zipkin.erl

```diff
@@ -78,6 +80,15 @@ zipkin_span(Span, LocalEndpoint) ->
        <<"tags">> => to_tags(Span#span.attributes) %% TODO: merge with oc_tags?
      }.
 
+format_trace_id(TraceId, tracecontext) ->
+    iolist_to_binary(io_lib:format("~32.16.0b", [TraceId]));
+
+format_trace_id(TraceId, b3) ->
+    iolist_to_binary(io_lib:format("~.16.0b", [TraceId]));
```

Comment:
I wasn't sure how to use the format lib, but not specifying the width made it so if it was 16 chars it wouldn't get padded with 0s.